src/tools/storybook-addon-code/utils/codepen.js

```diff
@@ -0,0 +1,30 @@
+const system = (process.env.STORYBOOK_SYSTEM || 'ec').toLowerCase();
+const baseUrl =
+  ['ec.europa.eu', 'localhost'].indexOf(window.location.hostname) !== -1
+    ? `https://v3--europa-component-library.netlify.app/playground/${system}`
+    : window.location.origin + window.location.pathname;
+
+const replaceLogo = (code) =>
+  code.replace(
+    /dist\/images\/logo([\d-az-]*)\.([\da-z]*)\.svg/gi,
+    `${baseUrl}images/logo/logo$1.svg`
```

Comment:
baseUrl is not added to the logo path on codepen, img is broken: `src="static/media/logo--en.5055ef4f.svg"`

Reply:
Yes, I tried to find a way to solve this within a few hours, but couldn't. V2 has the same issue.
I tried a few ideas, but I haven't figured how to work with the webpack-generated hashes consistently

Reply:
i.e. I am aware of this issue but haven't found a solution yet

Reply:
Sorry, I couldn't manage to solve the issue with display of SVG images on codepen.io domain.
I did manage to solve the issues with paths, but not with the cross-domain resource sharing limitations.
I spent the day trying to solve the issue with proxy server, but couldn't, so created https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1908 as a follow-up in order to not keep this pull request blocked for too long.